client/src/components/App/AppToolbar/AppToolbar.tsx

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
-import { AppBar, Toolbar } from '@material-ui/core';
-import { Typography } from '@material-ui/core';
-import Tooltip from '@material-ui/core/Tooltip';
+import { AppBar, Button, Toolbar, Typography, Tooltip } from '@material-ui/core';
+import { Home, SupervisorAccount } from '@material-ui/icons';
+import { useLocation } from 'react-router-dom';
 
 import IsActiveToggle from 'commons/IsActiveToggle/IsActiveToggle';
+import UserType from 'models/enums/UserType';
+import { landingPageRoute, usersManagementRoute } from 'Utils/Routes/Routes';
 
 import useAppToolbar from './useAppToolbar';
 
 const toggleMessage = 'מה הסטטוס שלך?';
 
 const AppToolbar: React.FC = (): JSX.Element => {
     const { user, isActive, setUserActivityStatus, classes, countyDisplayName } = useAppToolbar();
+    const location = useLocation();
 
     return (
         <AppBar className={classes.appBar} position='static'>
             <Toolbar >
-                <div className={classes.logoTitle}>
+                <div className={classes.rightSection}>
                     <img alt='logo' src='./assets/img/logo.png' width={48} height={48} />
                     <Typography variant='h4' >אבן יסוד</Typography>
+                    {
+                        (user.userType === UserType.ADMIN || user.userType === UserType.SUPER_ADMIN) &&
+                        (location.pathname === landingPageRoute || location.pathname === usersManagementRoute)
```

Comment:
```suggestion
                        (user.userType !== UserType.SIMPLE_USER) &&
                        (location.pathname !== investigationRoute)
```

Reply:
routes and rules may be added...

Reply:
If more user types will be added they will also be able to access these buttons automatically. Do you still think this should be a "blacklist" approach instead of a "whitelist" one?

Reply:
Same argument for adding more routes